pkg/destroy/ibmcloud/dns.go

```diff
@@ -49,7 +57,54 @@ func (o *ClusterUninstaller) listDNSRecords() (cloudResources, error) {
 	return cloudResources{}.insert(result...), nil
 }
 
+// listDNSSvcDNSRecords lists DNS Services DNS records for the cluster
+func (o *ClusterUninstaller) listDNSSvcDNSRecords() (cloudResources, error) {
+	o.Logger.Debug("Listing DNS Services DNS records")
+	ctx, cancel := o.contextWithTimeout()
+	defer cancel()
+
+	result := []cloudResource{}
+	moreData := true
+	var viewedResourceRecords int64
+	for moreData {
```

Comment:
could you change this to something a bit more descriptive(i.e. resourceRecordsRemaining)?

Reply:
Sure, I can do that

Reply:
Updated